tests/integ/test_local_mode.py

```diff
@@ -230,6 +251,53 @@ def test_tf_local_data_local_script():
         fcntl.lockf(local_mode_lock, fcntl.LOCK_UN)
 
 
+def test_local_mode_serving_from_s3_model(sagemaker_local_session, mxnet_model):
+    local_mode_lock_fd = open(LOCK_PATH, 'w')
+    local_mode_lock = local_mode_lock_fd.fileno()
+
+    model_data = mxnet_model.model_data
+    boto_session = sagemaker_local_session.boto_session
+    default_bucket = sagemaker_local_session.default_bucket()
+    uploaded_data = tar_and_upload_dir(boto_session, default_bucket,
+                                       'test_mxnet_local_mode', '', model_data)
+
+    s3_model = MXNetModel(model_data=uploaded_data.s3_prefix, role='SageMakerRole',
+                          entry_point=mxnet_model.entry_point, image=mxnet_model.image,
+                          sagemaker_session=sagemaker_local_session)
```

Comment:
Is it necessary to explicitly pass a LocalSession here, or is that only for our tests?

Reply:
It used to be required, now its only for our tests.